### DISCUSSION TITLE
Zoom.Zoom-preinstall.ps1

### DISCUSSION THREAD

**Discussion by MaximSidorchuk** [Romanitho/Winget-AutoUpdate#521] [MODs]
1. Add registry keys to enable auto-updates for Zoom. The App should update itself, instead of WAU.
2. When WAU update is triggered for Zoom, check before updating Zoom that the user is not joined to any meeting right now. If the meeting is running, skip the update this time.

[Zoom.Zoom-preinstall.ps1.zip](https://github.com/Romanitho/Winget-AutoUpdate/files/13647372/Zoom.Zoom-preinstall.ps1.zip)

**Comment by Romanitho** [comment]
Thanks :)
